src/test/suite/connectionController.test.ts

```diff
@@ -2,13 +2,16 @@ import sinon from 'sinon';
 import type { SinonStub } from 'sinon';
 import util from 'util';
 import * as vscode from 'vscode';
-import { afterEach, beforeEach } from 'mocha';
+import { afterEach, beforeEach, describe } from 'mocha';
```

Comment:
We use `suite` instead of `describe`. You don't need to import `suite` to use it.